src/phases/egg-hatch-phase.ts

```diff
@@ -12,12 +12,17 @@ import { Mode } from "#app/ui/ui.js";
 import i18next from "i18next";
 import SoundFade from "phaser3-rex-plugins/plugins/soundfade";
 import * as Utils from "#app/utils.js";
+import { EggLapsePhase } from "./egg-lapse-phase";
+import { EggHatchData } from "#app/data/egg-hatch-data.js";
```

Comment:
```suggestion
import { EggLapsePhase } from "./egg-lapse-phase";
import { EggHatchData } from "#app/data/egg-hatch-data";
```
NIT: Same comment as above